src/main/java/world/bentobox/bentobox/BentoBox.java

```diff
@@ -463,6 +463,10 @@ public boolean loadSettings() {
             getPluginLoader().disablePlugin(this);
             return false;
         }
+
+        log("Saving default panels...");
+        this.saveResource("panels/language_panel.yml", false);
```

Comment:
This begs a question - if we used false as the second parameter then if this panel ever needs to be updated by us, e.g., new code, then we will have to have admins remove the old panel yml file when updating. If however, we use true, then we will overwrite the file and ensure that the latest file is used with the latest code. Do you think there's ever going to be a need for admins to adjust these panels? And even if they did, due to code updates, their old yml may not work with the new release, so it'd be better to overwrite it. What do you think?

Reply:
I do not think it is necessary to overwrite them. Otherwise how admins will be able to customize their menus if we overwrite them always.

Reply:
Yeah, good point. The only thing then going forward is that instructions for upgrading will need to include directions to delete the old panel file (or move it) if the change is affected by it. During my development of the team GUI I've had a number of times I forgot to delete it and wondered why I was getting weird operations.